packages/affiliates/apps/DevMiningRewards.js

```diff
@@ -101,6 +105,94 @@ const App = (env) => async (params) => {
   };
 };
 
+const v2 = async (env, params, DevMining) => {
```

Comment:
this is in the dev mining application. There are 2 blocks of code that configure things for v1 (tvm based dev mining) and now v2 (tvl based dev mining). This new block is almost a copy and paste of the v1 block, except it removes any mention of synthetic tokens.